templates/templates/docs/discourse.html

```diff
@@ -0,0 +1,108 @@
+
+{% extends "templates/base.html" %}
+
+{% block title %} {{ document.title }} | Server documentation{% endblock %}
```

Comment:
not that I have looked at all the code, but should ' | Server documentation' be in this file?

Reply:
Your right, copy and paste error. Done